spec/plans/report.fmf

```diff
@@ -262,3 +237,29 @@ description:
         to the execute step plan run directory. The ``log`` key
         will be a list of such paths, even if there is just a single
         log.
+
+    example: |
+        .. code-block:: yaml
```

Comment:
Should be removed, the story `example` key is automatically rendered as `yaml`. Now the directive is shown in the generated docs.

Reply:
@psss done !

Reply:
Good!